bgpd/bgp_updgrp_adv.c

```diff
@@ -316,7 +316,7 @@ static int subgroup_coalesce_timer(struct thread *thread)
 		zlog_debug("u%" PRIu64 ":s%" PRIu64
 			   " announcing routes upon coalesce timer expiry(%u ms)",
 			   (SUBGRP_UPDGRP(subgrp))->id, subgrp->id,
-			   subgrp->v_coalesce),
+			   subgrp->v_coalesce);
```

Comment:
how it worked? :/

Reply:
It pulled the next line into a C comma statement...  I'm more surprised it didn't get a warning from GCC, since it does warn about misindented if blocks in other situations.